jupiterone/util.go

```diff
@@ -5,3 +5,13 @@ import "strings"
 func removeCRFromString(s string) string {
 	return strings.ReplaceAll(s, "\r", "")
 }
+
+func interfaceSliceToStringSlice(l []interface{}) []string {
```

Comment:
Could you help me understand this func? and the syntax of `l []interface{})`?

Reply:
`interface{}` is the empty interface, which says that the underlying type implements at least zero methods. Every type implements at least zero methods, so `interface{}` can hold values of any type. In newer Go with generics, there is a new type `any`, which is essentially the same thing.

This method goes through the slice of `interface{}` and asserts they are strings, and returns the resolved `[]string`. If one of them wasn't a string it would panic.

https://go.dev/tour/methods/14